src/mmw/js/src/core/modals/models.js

```diff
@@ -30,8 +30,35 @@ var ShareModel = Backbone.Model.extend({
     }
 });
 
+var AlertTypes = {
+    info: {
+        alertHeader: 'Information',
+        alertIcon: '\"fa fa-info-circle\"',
+        dismissLabel: 'Okay'
+    },
+    warn: {
+        alertHeader: 'Warning',
+        alertIcon: '\"fa fa-exclamation-triangle\"',
+        dismissLabel: 'Okay'
+    },
+    error: {
+        alertHeader: 'Error',
+        alertIcon: '\"fa fa-ban\"',
```

Comment:
Prefer not escaping strings here, and quoting them in the template. Also, since they all have `fa`, consider putting that in the template for less redundancy:

```js
error: {
    alertHeader: 'Error',
    alertIcon: 'fa-ban',
    dismissLabel: 'Okay',
}
```

```jinja
<i class="fa {{ alertIcon }}"></i>
```